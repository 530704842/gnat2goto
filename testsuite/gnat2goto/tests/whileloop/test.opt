ALL XFAIL CBMC fails with invariant failure due to op.type().id()!=ID_bool in not_exprt
