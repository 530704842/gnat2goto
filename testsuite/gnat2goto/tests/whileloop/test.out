[1] : SUCCESS
[2] : SUCCESS
[overflow.1] arithmetic overflow on signed + in whileloop__a + whileloop__c: SUCCESS
[overflow.2] arithmetic overflow on signed - in whileloop__a - whileloop__c: SUCCESS
VERIFICATION SUCCESSFUL
